src/osgEarthSymbology/TextSymbol

```diff
@@ -239,6 +243,7 @@ protected:
     optional<std::string>       _predefinedOrganisation;
     optional<double>            _predefinedOrganisationMargin;
     optional<bool>              _backEarthCull;
+    optional<NumericExpression> _textPriority;
```

Comment:
**Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority**

replace textPriority by priorityFine everywhere